packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -572,6 +599,35 @@ const formatJsonValue = (value) => {
 	return value;
 };
 
+const setPresetValues = (data: CiemssPresetTypes) => {
+	if (data === CiemssPresetTypes.Normal) {
+		knobs.value.numSamples = qualityValues.numSamplesToSimModel;
+		knobs.value.solverMethod = qualityValues.method;
+		knobs.value.maxiter = qualityValues.maxiter;
+		knobs.value.maxfeval = qualityValues.maxfeval;
+	}
+	if (data === CiemssPresetTypes.Fast) {
+		knobs.value.numSamples = speedValues.numSamplesToSimModel;
+		knobs.value.solverMethod = speedValues.method;
+		knobs.value.maxiter = speedValues.maxiter;
+		knobs.value.maxfeval = speedValues.maxfeval;
+	}
+};
+
+const speedValues = {
+	numSamplesToSimModel: 1,
+	method: CiemssMethodOptions.euler,
+	maxiter: 0,
+	maxfeval: 1
+};
+
+const qualityValues = {
+	numSamplesToSimModel: 100,
+	method: CiemssMethodOptions.dopri5,
+	maxiter: 5,
+	maxfeval: 25
+};
```

Comment:
May want to consider adding `Object.freeze` to the presets to lower the risk of someone accidentally changing things

```
const speedValues = Object.freeze({
  blahblah: 123
})
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze

Reply:
oh neat! I'm going to bring this to the other two as well.